### ISSUE TITLE
gulp watch doesn't update on scss changes

### ISSUE THREAD

**Issue by blakejwc** [vecnatechnologies/quick-sip#13]
With the default options and some standard options:
```javascript
var gulp = require('gulp'),
    brec = require('brec-tables'),
    buildProcess = require('quick-sip')(gulp, {
      browserify: {
        transforms: [
          {transform: 'aliasify', options: {global: true}},
          'hbsfy'
        ]
      },
      styles: {
        includes: [brec.base.sass, brec.tables.sass]
      },
      copy: {
        excludes: 'js|hbs|scss'
      }
    });
```
Directory structure:
```
    ├─ .gitignore
    ├─ package.json
    ├─ gulpfile.js
    ├─ dist
    └─ app
        ├─ app.scss
        ├─ app.js
        ├─ router.js
        ├─ index.html
        └─ home
            ├─ homeView.js
            ├─ home-template.hbs
            └─ _home.scss
```

**Comment by kentmw** [comment]
Also, is it calling the right task?

https://github.com/vecnatechnologies/quick-sip/blob/master/tasks/watch.js#L49

has:
```
gulp.watch(options.styles.src, ['build-styles']);
```
shouldn't it be:
```
gulp.watch(options.styles.src, [options.taskPrefix + 'build-styles']);
```

**Comment by mandragorn** [comment]
that looks correct.  I assume your watch edit above is the same path that build-styles is using (if they can be linked somehow that would be ideal).  In case we change how build-styles works.

**Comment by mandragorn** [comment]
+1 - beiyi just ran into this issue.

**Comment by mandragorn** [comment]
Fixed by #18

**Comment by blakejwc** [comment]
So I discovered that quick-sip doesn't have a default for options.styles.src, so the generator-torso-brec project won't work out of the box.

Additionally, it requires that options.styles.src be defined in the gulpfile.  I'm going to change the default options.styles.src to be options.src to handle this bit of inconsistency in my pull request for issue #14.
